src/bootstrap/src/utils/execution_context.rs

```diff
@@ -161,7 +161,7 @@ pub struct DeferredCommand<'a> {
     command: &'a mut BootstrapCommand,
     stdout: OutputMode,
     stderr: OutputMode,
-    created_at: Location<'a>,
+    executed_at: &'a Location<'a>,
```

Comment:
FWIW, the function returns `&'static Location<'static>`, I think? So you could just use that. But it doesn't really matter.